app/phone/src/main/java/dev/jdtech/jellyfin/BasePlayerActivity.kt

```diff
@@ -24,22 +26,40 @@ abstract class BasePlayerActivity : AppCompatActivity() {
     override fun onResume() {
         super.onResume()
 
-        viewModel.player.playWhenReady = viewModel.playWhenReady
+        if (wasPip) {
+            wasPip = false
+        } else {
+            viewModel.player.playWhenReady = viewModel.playWhenReady
+        }
         hideSystemUI()
     }
 
     override fun onPause() {
         super.onPause()
-        viewModel.playWhenReady = viewModel.player.playWhenReady == true
-        viewModel.player.playWhenReady = false
+
+        if (isInPictureInPictureMode) {
+            wasPip = true
+        } else {
+            viewModel.playWhenReady = viewModel.player.playWhenReady == true
+            viewModel.player.playWhenReady = false
+        }
     }
 
     override fun onStop() {
         super.onStop()
 
+        viewModel.playWhenReady = false
+        viewModel.player.playWhenReady = false
         mediaSession.release()
```

Comment:
Adding this to `onStop()` fixed the issue that was happening with pip.

Can you try it out with this change? @jarnedemeulemeester 

```suggestion
		wasPip = false
        mediaSession.release()
```